src/commands/add.py

```diff
@@ -0,0 +1,43 @@
+from src.shell import Shell
+from src.utils import Utils, Geometry, Crypt
+from src.password   import Password
+from getpass        import getpass
+
+class Add:
+
+    @staticmethod
+    def do(shell: Shell, line: str):
+        if shell.session.name is not None:
```

Comment:
Here I would write 
```
if shell.session.name is None:
    shell.help_add()
    return
# rest of the code
```
This would make the if / else statement clearer (instead of the else at the very bottom of the method)